src/Frontend/Themes/Bootstrap/src/Js/lib/share-btns.js

```diff
@@ -0,0 +1,123 @@
+;(function ( $, window, document, undefined ) {
```

Comment:
It's from the jQuery plugin boilerplate:

```		// undefined is used here as the undefined global variable in ECMAScript 3 is
		// mutable (ie. it can be changed by someone else). undefined isn't really being
		// passed in so we can ensure the value of it is truly undefined. In ES5, undefined
		// can no longer be modified.```